src/main/kotlin/io/polygon/kotlin/sdk/rest/Quotes.kt

```diff
@@ -58,11 +60,12 @@ data class QuotesResponse(
     override val results: List<Quote>? = null
 ) : Paginatable<Quote>
 
+@ExperimentalSerializationApi
 @Serializable
 data class Quote(
     @SerialName("ask_exchange") val askExchange: Int? = null,
-    @SerialName("ask_price") val askPrice: Double? = null,
+    @JsonNames("ask", "ask_price") val askPrice: Double? = null,
     @SerialName("bid_exchange") val bidExchange: Int? = null,
-    @SerialName("bid_price") val bidPrice: Double? = null,
+    @JsonNames("bid", "bid_price") val bidPrice: Double? = null,
```

Comment:
`v3/snapshot` uses `{last_quote:{ask: 100}}` whereas `v3/quotes` uses `{ask_price: 100}`. We can use this experimental `JsonNames` feature to cover both and reuse the class. Otherwise I will need to create a new `SnapshotQuote` class.

Reply:
This didn't work because you can't have default values in this case. I also attempted to refactor this to use a `quotesCommon` object for shared fields but didn't like the result. Lastly, I tried using interfaces, but it doesn't actually save any verbosity (still need separate quotes objects), and we don't use these interchangeable, so the interface doesn't give us any functional benefits.